OpenRA.Mods.Common/Traits/Conditions/GrantConditionOnDeployWithCharge.cs

```diff
@@ -151,7 +154,15 @@ Order IIssueDeployOrder.IssueDeployOrder(Actor self, bool queued)
 		public void ResolveOrder(Actor self, Order order)
 		{
 			if (order.OrderString == "DeployWithCharge")
```

Comment:
```suggestion
			if (order.OrderString != "DeployWithCharge")
			    return;
```
For improved readability.